webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/resource/annotation_log.js

```diff
@@ -52,13 +52,13 @@ var AnnotationLog = (function(window, undefined) {
             log: _action,
```

Comment:
Please no changes to the brat JS files. They should be as much in-sync with the official brat as possible to facilitate exchanging code between WebAnno and brat.